client/src/components/categorical/value/index.js

```diff
@@ -480,6 +480,37 @@ class CategoryValue extends React.Component {
           CHART_MARGIN
         : globals.leftSidebarWidth - otherElementsWidth;
 
+    let stackedBins;
+    let histogramBins;
+
+    // If coloring by and this isn't the colorAccessor and it isn't being edited
```

Comment:
Unsure if the comment reflects the condition, since the comment says "if coloring by" while the condition is `&& !isColorBy &&`, which seems to mean it's NOT coloring by?

Thank you!

Reply:
Mistake of poor verbiage, I meant coloring by in the general sense that _something_ is being colored by not that the present value is being colored by.  I'll try to clear this up with better wording

Reply:
Ah thank you! The misunderstanding is likely because I'm not familiar with the code base yet 😆 But thanks for the clarification!